lib/fib_sink_vb_impl.cc

```diff
@@ -346,14 +400,17 @@ namespace gr {
             }
             case FIB_SI_EXTENSION_SERVICE_COMP_LABEL:
               memcpy(label, &data[5], 16);
-              GR_LOG_DEBUG(d_logger, format("[service component label] %s") % label);
+              GR_LOG_DEBUG(d_logger,
+                           format("[service component label] %s") % label);
               break;
             case FIB_SI_EXTENSION_DATA_SERVICE_LABEL:
               memcpy(label, &data[5], 16);
-              GR_LOG_DEBUG(d_logger, format("[data service label]: %s") % label);
+              GR_LOG_DEBUG(d_logger,
```

Comment:
My happiness about this kind of line breaks is limited, but it's up to your coding style, for sure.